### DISCUSSION TITLE
GET  mercure-info 501  index-Ch_V4_Dp.js:64   (Not Implemented)

### DISCUSSION THREAD

**Discussion by xyychenchen** [shlinkio/shlink#2262] [Help wanted] [answered]
configuration：
Server: [v4.2.5](https://github.com/shlinkio/shlink/releases/v4.2.5) - Client: [v4.2.2](https://github.com/shlinkio/shlink-web-client/releases/v4.2.2)
nginx/1.22.1
PHP8.2

the web client can be accessed normally but when I turn on F12 on my Windows Chrome browser I see this error notification , how to solve it my friend? 

GET https://link.kkberry.com/rest/v3/mercure-info 501 
index-Ch_V4_Dp.js:64 
 (Not Implemented)

**Comment by acelaya** [comment, marked as answer]
That's expected. It's how Shlink tells the client that your instance is not integrated with mercure
